mappings/net/minecraft/client/render/entity/EntityRenderDispatcher.mapping

```diff
@@ -17,6 +17,8 @@ CLASS net/minecraft/class_898 net/minecraft/client/render/entity/EntityRenderDis
 	METHOD method_17145 register (Lnet/minecraft/class_1299;Lnet/minecraft/class_897;)V
 	METHOD method_23165 renderFire (Lnet/minecraft/class_4587;Lnet/minecraft/class_4597;Lnet/minecraft/class_1297;)V
 		ARG 1 matrices
+	METHOD method_23167 registerRenderers (Lnet/minecraft/class_918;Lnet/minecraft/class_3296;)V
+	METHOD method_23168 squaredDistanceToCamera (Lnet/minecraft/class_1297;)D
```

Comment:
`getSquaredDistanceToRenderer`

Reply:
This is just an overload to an already existing method which takes the three coords.
Should I rename both?

Reply:
I suggest `getSquaredDistanceToCamera` as `camera` is more accurate than `renderer`

Reply:
👍